.ruby-version

```diff
@@ -1 +1 @@
-2.1.0
+ruby-2.2.2
```

Comment:
Is the `ruby-` needed, or can we just do `2.2.2`  I use rvm, which is fine with this format, but I think rbenv doesn't like the `ruby-` prefix.